index.html

```diff
@@ -0,0 +1,250 @@
+       <!-- Section 2 started -->
+       <section class="section2">
+        <h2 class="h23">I’m Aziz Glad to see you!</h2>
+         <h4  class="paragraph">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
+  <h5 class="ets" >ETS CONNECT</h5>
+ <ul class="social-menue">
+  <li><img src="images/twitter.PNG" alt="abc"  class="lsls"></li>
+  <li><img src="images/linkedinicon.PNG" alt="abc" class="lsls"></li>
+  <li> <img src="images/medium.PNG" alt="abc" class="lsls"></li>
+  <li> <img src="images/github.PNG" alt="abc" class="lsls"></li>
+  <li><img src="images/angellist.PNG" alt="abc" class="lsls"></li>
+ </ul>
+          
+ </section>
+   <!-- section 2 closed -->
+
+   <section class="abd">
+    <div class="left">
+      
+      <img src="images/a.png" alt="resimage">
+
+    </div>
+
+    <div class="right">        
+      <h2 class="tonic">Tonic</h2>
+      <ul>
+        <li class="canopli">
+           <a href="/someurl" class="sec3rlink31">CANOPY</a>
+            <b class="sec3p">.</b>
+            <a href="/someurl" class="sec3rlink31">Backen Dev</a>
+             <b class="sec3p">.</b>
+           <a href="/someurl" class="sec3rlink31">2015</a>
+         </li>
+      </ul>
+<p class="paragraph1"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+<ul class="ad">
+  <li class="canopli2">
+       <a href="/someurl" class="sec3rlink32">HTML</a>
+       <a href="/someurl" class="sec3rlink32">CSS</a>
+       <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+   </li>
+</ul>
+<button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+</div>
+  </section>
+<!-- Section 3 completed -->
+
+<!-- Section 4 started -->
+<section class="row3">
+  <div class="r3left">
+    <h2 class="tonic">Multi-Post Stories</h2>
+    <ul>
+      <li class="canopli">
+         <a href="/someurl" class="sec3rlink31">FACEBOOK</a>
+          <b class="sec3p">.</b>
+          <a href="/someurl" class="sec3rlink31">Full Stack Dev</a>
+           <b class="sec3p">.</b>
+         <a href="/someurl" class="sec3rlink31">2015</a>
+       </li>
+    </ul>
+<p class="paragraph1">Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+
+<ul class="ad">
+<li class="canopli2">
+     <a href="/someurl" class="sec3rlink32">HTML</a>
+     <a href="/someurl" class="sec3rlink32">Ruby on rails</a>
+     <a href="/someurl" class="sec3rlink32">CSS</a>
+     <a href="/someurl" class="sec3rlink32">JAVASCRIPT</a>
+ </li>
+</ul>
+<button type="submit" class="bb sec3rbutton sec3rlink4"><span class="d">See Projects</span></button>
+  </div>
+  <div class="right">        
+    <img src="images/b.png" alt="">
+  </div>
```

Comment:
Kindly make all the works resemble the Figma design for both desktop and mobile 🤗 as described previously.